kn@1.7.rb

```diff
@@ -0,0 +1,39 @@
+require "fileutils"
+
+class Kn < Formula
```

Comment:
Should it be `KnAT17` ?

also, seeing version mismatch in https://github.com/knative/homebrew-client/blob/main/kn%401.6.rb#L6